grpc/src/main/java/com/netflix/conductor/grpc/AbstractProtoMapper.java

```diff
@@ -49,7 +49,7 @@
 import java.util.stream.Collectors;
 import javax.annotation.Generated;
 
-@Generated("com.github.vmg.protogen.ProtoGen")
+@Generated("com.netflix.conductor.annotationsprocessor.protogen")
 public abstract class AbstractProtoMapper {
```

Comment:
Sure thing, that is a good suggestion. One thought around keeping this is that we can see the changes in the code generated when it is checked into version control. Let me know if you have any thoughts on that, if not we can ignore this file and I can make that change.

Reply:
I don't have a strong opinion on keeping vs removing the file. I see your point, since the file is generated by conductor project now (annotation-processor) versus an external library, it will help keeping it around.